liquibase-integration-tests/src/test/groovy/liquibase/command/core/UpgradeChecksumVersionIntegrationTest.groovy

```diff
@@ -19,16 +19,16 @@ import spock.lang.Unroll
 @LiquibaseIntegrationTest
 class UpgradeChecksumVersionIntegrationTest extends Specification{
     @Shared
-    private DatabaseTestSystem mysql = (DatabaseTestSystem) Scope.getCurrentScope().getSingleton(TestSystemFactory.class).getTestSystem("mysql")
+    private DatabaseTestSystem h2 = (DatabaseTestSystem) Scope.getCurrentScope().getSingleton(TestSystemFactory.class).getTestSystem("h2")
```

Comment:
We don't need mysql here as h2 is a lot faster to start.

Reply:
I wonder how many other integration tests use a heavy database that could use H2. Probably most of them.